### ISSUE TITLE
Support triggering a NVDA profile along with a WebModule

### ISSUE THREAD

**Issue by JulienCochuyt** [accessolutions/WebAccessForNVDA#17]
Now that profiles can be manually triggered in NVDA, it should be fairly simple to fit the needs expressed in nvaccess/nvda#4442.
